external/radiation/radiation/wrapper_api.py

```diff
@@ -223,3 +253,24 @@ def _solar_hour(time: cftime.DatetimeJulian) -> float:
         + time.minute / MINUTES_PER_HOUR
         + time.second / (MINUTES_PER_HOUR * SECONDS_PER_MINUTE)
     )
+
+
+def _get_forecast_time_index(
+    current_time: cftime.DatetimeJulian,
+    init_time: cftime.DatetimeJulian,
+    timestep_seconds: float,
+) -> int:
+    """Get integer index of forecast time, since initial time """
+    forecast_elapsed_seconds = (
+        current_time + timedelta(seconds=timestep_seconds) - init_time
+    ) / timedelta(seconds=1)
+    return int(forecast_elapsed_seconds / timestep_seconds)
```

Comment:
```suggestion
    forecast_elapsed_seconds = (current_time - init_time).total_seconds()
    return int(forecast_elapsed_seconds / timestep_seconds) + 1
```

IMO a bit more readable.

Reply:
Done